css/css-flexbox/flexbox-iframe-intrinsic-size.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com"/>
+<link rel="help" href="https://drafts.csswg.org/css-flexbox/#algo-main-item" title="Part E">
+<link rel="help" href="https://www.w3.org/TR/css-sizing-3/#intrinsic-sizes" title="For boxes with no preferred aspect ratio">
```

Comment:
Could you add a "meta assert" explaining the purpose of the test?

Reply:
I thought it was enough with the <p> explaining the result but yeah I can add it.